<launch>
    <!--<node pkg="laser_line_extraction" type="line_detector_ekf_node" name="line_detector_ekf_node" output="screen">-->
    <!--<param name="rate" value="30"/>-->
    <!--<param name="cov_x" value="0.005"/>-->
    <!--<param name="cov_y" value="0.03"/>-->
    <!--<param name="cov_yaw" value="0.02"/>-->

    <!--</node>-->

    <!--<node pkg="robot_pose_ekf" type="robot_pose_ekf" name="triangle_pose_ekf" output="screen">-->
    <!--<param name="output_frame" value="base_triangle"/>-->
    <!--<param name="base_footprint_frame" value="base_link"/>-->
    <!--<param name="freq" value="30.0"/>-->
    <!--<param name="sensor_timeout" value="1.0"/>-->
    <!--<param name="odom_used" value="true"/>-->
    <!--<param name="imu_used" value="true"/>-->
    <!--<param name="vo_used" value="false"/>-->

    <!--<param name="inverse_tf" value="true"/>-->
    <!--<remap from="odom" to="triangle_odom" />-->
    <!--</node>-->

    <node name="line_detector_node" pkg="laser_line_extraction" type="line_detector_node">
        <rosparam command="load" file="$(find laser_line_extraction)/launch/params.yaml" />
        <param name="max_search_radius" value="0.7" type="double"/>
        <param name="min_intensity" value="200" type="double"/>


    <rosparam>
      markers_array :
        - x : 4.135
          y : -1.622
        - x : 4.177
          y : -1.245
        - x : 4.189
          y : -0.867
      target_pose :
        x : 0.0
        y : -0.0
        yaw : 0.0

        </rosparam>

    </node>

    <!--<node name="tf1" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 1 odom base_link 20" />-->
    <!--<node name="tf2" pkg="tf" type="static_transform_publisher" args="0.1 0 0 0 0 0 1 base_link base_laser 20" />-->
    <!--<node name="tf3" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 1 map odom 20" />-->

    <node name="tf1" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 1 odom base_link 20"/>
    <node name="tf2" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 1 base_link base_laser 20"/>
    <node name="tf3" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 1 map odom 20"/>



</launch>

